### PR TITLE
fix(CupertinoSlidingSegmentedStyle): change height to minheight

### PR DESCRIPTION
**Pull request by Asfiroth**
GitHub Issue (If applicable):  #1021 

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

Using TabBar control with CupertinoSlidingSegmentedStyle renders ui with fixed height

## What is the new behavior?

Using TabBar control with CupertinoSlidingSegmentedStyle renders ui with adaptable height

Screenshot 2024-02-01 at 14 51 18

## PR Checklist

Please check if your PR fulfills the following requirements:
- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Tested the changes where applicable:
	- [ ] UWP
	- [ ] WinUI
	- [x] iOS
	- [x] Android
	- [ ] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	- [ ] [General Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc)
	- [ ] [Controls Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc/controls)
	- [ ] [Extensions Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc/helpers)
	- [ ] [controls-styles.md](https://github.com/unoplatform/uno.toolkit.ui/blob/main/doc/controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/uno.toolkit.ui/blob/main/doc/lightweight-styling.md)
- [ ] [Runtime Tests and/or UI Tests](https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by Asfiroth** [comment]
According to logs, build is failing due to: `error MSB4044: The "AndroidSignPackage" task was not given a value for the required parameter "KeyStore". [D:\a\1\s\samples\Uno.Toolkit.Samples\Uno.Toolkit.Samples.Mobile\Uno.Toolkit.Samples.Mobile.csproj::TargetFramework=net7.0-android]`

File: src/library/Uno.Toolkit.Cupertino/Styles/Controls/SlidingSegmentedControl.xaml [hunk 1]
```diff
@@ -406,17 +408,16 @@
 			<Setter.Value>
 				<DataTemplate>
 					<toolkit:ElevatedView Width="{Binding RelativeSource={RelativeSource Mode=TemplatedParent}, Path=TemplateSettings.SelectionIndicatorWidth}"
-										  Height="28"
+					                      Margin="{StaticResource CupertinoSlidingSegmentedItemMargin}"
 										  Background="Transparent"
-										  CornerRadius="8"
+										  CornerRadius="{CustomResource CupertinoSlidingSegmentedItemCornerRadius}"
```
Comment (kazo0) [inline src/library/Uno.Toolkit.Cupertino/Styles/Controls/SlidingSegmentedControl.xaml:413]:
```suggestion
										  CornerRadius="{StaticResource CupertinoSlidingSegmentedItemCornerRadius}"
```

**Review by kazo0 — APPROVED** [review]

File: src/library/Uno.Toolkit.Cupertino/Styles/Controls/SlidingSegmentedControl.xaml (hunk 1 above)
Comment (Asfiroth) [reply src/library/Uno.Toolkit.Cupertino/Styles/Controls/SlidingSegmentedControl.xaml]: Thanks, I didn't noticed it was set as `CustomResource`

**Comment by Asfiroth** [comment]
Still having this issue `The "AndroidSignPackage" task was not given a value for the required parameter "KeyStore".` @kazo0

**Comment by kazo0** [comment]
@Asfiroth sorry about that :) trying to fix the CI for forks [here](https://github.com/unoplatform/uno.toolkit.ui/pull/1029)

**Comment by Asfiroth** [comment]
oh, no worries @kazo0 just letting you know haha

**Comment by sasakrsmanovic** [comment]
Thanks for contributing @Asfiroth !!

**Comment by Asfiroth** [comment]
I'm not quite sure pipeline failed cause it was expecting a file `/Users/runner/work/1/s/build/TestResult-ios-UWP-Automated.xml: No such file or directory` or something related to tests 🤔

**Pull request merged**